### PR TITLE
refactor: initial commit for password-policy-validation

### PR DESCRIPTION
**Pull request by Yukesh4791**

### CODE REVIEW

File: PasswordReset/PasswordReset/settings.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```
Comment (tverbeke) [inline PasswordReset/PasswordReset/settings.py]: does the comment above (don't run with debug turned on in production) apply here?

File: PasswordReset/app/pwdmanager.py [hunk 2]
```diff
@@ -58,7 +65,59 @@ def __kerberos_init():
...
+#
+#######################################################   
+
+############### Validate Password Policy #####################
+    def __vaidate_password(self, new_password):
+        policy = PasswordPolicy.from_names(
+            length=8,  # min length: 8
+            uppercase=2,  # need min. 2 uppercase letters
+            numbers=2,  # need min. 2 digits
+            special=1,  # need min. 2 special characters
```
Comment (tverbeke) [inline PasswordReset/app/pwdmanager.py:112]: comment not in line with value :-)

File: PasswordReset/app/pwdmanager.py [hunk 3]
```diff
@@ -58,7 +65,59 @@ def __kerberos_init():
...
+            nonletters=0,  # need min. 2 non-letter characters (digits, specials, anything)
+        )
+        
+        validate = policy.test(new_password)
+        if validate:
+            raise SetPasswordFailed("Cannot update your password. Your password should have atleast {0}".format(validate))
```
Comment (tverbeke) [inline PasswordReset/app/pwdmanager.py]: typo: at least (add space)

File: PasswordReset/PasswordReset/settings.py (hunk 1 above)
Comment (Yukesh4791) [reply PasswordReset/PasswordReset/settings.py]: it should apply here :p Will make the change.

File: PasswordReset/app/pwdmanager.py (hunk 2 above)
Comment (Yukesh4791) [reply PasswordReset/app/pwdmanager.py:112]: comment was from the initial developer and I just changed the value as per our need :D

**Pull request merged**